tests/e2e_tests

```diff
@@ -36,14 +37,33 @@ do
         echo "" >> ../error.log
     fi
     cd ..
-done
 
-if [ $failure = "true" ]
-then
-    echo "One of the e2e tests failed, check the logs";
-    cat error.log
-    exit 1
-fi
+    if [ "$failure" = "true" ]
+    then
+        exit 255
```

Comment:
why 255 rather than 1?

Reply:
255 tells to xargs to stop immediately if one of the process fails. If you use return status 1, it continues execution